composer.json

```diff
@@ -14,7 +14,7 @@
         "drupal/diff": "^1.0@RC",
         "drupal/digital_size_formatter": "~1.0-alpha1",
         "drupal/ds": "^3.0",
-        "drupal/facets": "1.0-alpha7",
+        "drupal/facets": "1.0-alpha8",
```

Comment:
It is very unfortunate that this was hardcoded to alpha8 instead of reverting to normal version tracking with `~1.0@alpha` :(

Because of this it has been missed that there has been a new Facets release on Monday :(

Let's update to the latest version.